force-app/main/default/classes/ProgramEngagementService.cls

```diff
@@ -43,4 +43,17 @@ public with sharing class ProgramEngagementService {
 
         return activeStages;
     }
+
+    public Map<String, String> getActiveStagesWithLabels() {
```

Comment:
Should this follow our normal naming convention maybe something like getActiveStagesByValue?